Source/Core/DolphinWX/Cheats/GeckoCodeDiag.cpp

```diff
@@ -192,6 +192,9 @@ void CodeConfigPanel::DownloadCodes(wxCommandEvent&)
 
 		while ((std::getline(ss, line).good()))
 		{
+			// Remove \r at the end of the line for files using windows line endings, std::getline only removes \n
+			line = StripSpaces(line);
+
 			// empty line
 			if (0 == line.size() || line == "\r" || line == "\n") // \r\n checks might not be needed
```

Comment:
Haha, could it be that the new line checks were some kind of workaround for the issue? With 2 windows style line endings directly after each other, there would be a line with \r.